cosmovisor/cmd/cosmovisor/cmd/help.go

```diff
@@ -24,13 +21,6 @@ func ShouldGiveHelp(arg string) bool {
 func DoHelp() {
 	// Not using the logger for this output because the header and footer look weird for help text.
 	fmt.Println(GetHelpText())
-	// Check the config and output details or any errors.
-	// Not using the cosmovisor.Logger in order to ignore any level it might have set,
-	// and also to not have any of the extra parameters in the output.
-	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: time.Kitchen}
-	logger := zerolog.New(output).With().Timestamp().Logger()
-	cfg, err := cosmovisor.GetConfigFromEnv()
-	cosmovisor.LogConfigOrError(logger, cfg, err)
```

Comment:
@ryanchristo has a good point.
> Running cosmovisor start prints the help text without showing the configuration errors. A user would have to run cosmovisor version to debug why cosmovisor is showing the help text before being able to run cosmovisor start successfully.

We should probably put this back in.

I know that printing errors during a `help` command is a little unorthodox, but I feel like it's very useful for in this case. I feel like it will almost always be desirable information, and when it's superfluous, it's not enough info to be annoying and get in the way. And when there aren't any errors, actually listing all the values seems very helpful too, specially since we didn't want to double up the documentation to include info about them in the help message itself.

Reply:
Let's get more consensus here. For me, printing error on help is a sign of a broken software. That's why I wrote in the description: _`cosmovisor help` should not print any error._

Reply:
My main concern is that running `cosmovisor start` without setting up the environment variables will print the help text and no errors. Running start without the environment variables set up should print errors; the user needs to know why cosmovisor did not start.

Reply:
Agree on that. But here, I want to agree on `help`. Personally, I'm against showing any errors in help.